### ISSUE TITLE
Accessibility: Missing keyboard navigability and focus

### ISSUE THREAD

**Issue by Menelion** [capy-ui/capy#56]
First of all, let me thank you for mentioning accessibility straight in the main Readme file for the repo, it's hugely appreciated. As a screen reader user and a software engineer myself, I'm in constant search of modern accessible solutions implementing native UIs for different platforms. Now to the issue itself.

### Steps to reproduce
* clone the repo in Windows.
* Open your Windows Terminal in PowerShell or whatever you use
* Do `zig build calculator`
* When the calculator is run, try to navigate with your keyboard using *Tab* or your arrow keys.
*Expected result*: The focus should be placed somewhere in the window on first run, probably in the edit field; Tab key should move through the buttons and other controls.
*Actual Result*: Nothing happens. As a screen reader user, I can only emulate mouse clicks which is far from being accessible and usable.

**Comment by zenith391** [comment]
You're right, I'll try to make a fix in the next few hours. Also, while reproducing the bug I noticed keyboard navigation was broken on  the Linux backend too so I'll fix that too

**Comment by zenith391** [comment]
So far I've been able to develop something basic but that works on Windows, but I'll have to expand my solution a bit more to allow support for arrow keys.
